### ISSUE TITLE
Reach end/start events not send on update call with 1.x

### ISSUE THREAD

**Issue by sconix** [utatti/perfect-scrollbar#695]
In 1.x it seems that the reach end/start events are not send on update call only when scrolling. By looking at the code the reach event sending is only send if the old position differs from the new position which is not the case when update is called in the init/manually. Is this a bug or intended behavior? 

I believe its a bug since the update code still has a comment '// Update top/left scroll to trigger events
' and as the updateScroll works now I don't see how it could send any events on manual update call.

If its intended behavior then is there any way to trigger those reach end/start events or check the status on init / after programmatic scroll without checking scroll position / geometry values manually.

**Comment by sconix** [comment]
I do agree that this is better than how it was in the 0.8.x. I was kinda thinking more like a parameter for update / init so that the scroll state event could be send if wanted or providing a function to do the checking. I just hate "duplicating" a code so would be nice if the check done by the library would be available as helper function etc. Since its really common that you relay on the reach end event that you might want to know the status after init / programmatic scrolls as well.

**Comment by utatti** [comment]
I agree that the new way can result in code duplication. I’ll see if I can add a helper.

**Comment by utatti** [comment]
I've added a `reach` property, which always watching the state of scroll reach. With this you do not need to check scroll properties manually.

For its usage, please refer to `events.html`. I've updated README too.

- https://github.com/utatti/perfect-scrollbar/blob/master/examples/events.html#L72

It will be released in the next version. I'm not sure when it comes, maybe soon 😄
